lib/broken_record/scanner.rb

```diff
@@ -30,11 +30,11 @@ def models_to_validate(model_name)
     end
 
     def load_all_active_record_classes
-      Dir.glob(Rails.root.to_s + '/app/models/**/*.rb').each { |file| require file }
+      Rails.application.eager_load!
       objects = Set.new
       # Classes to skip may either be constants or strings.  Convert all to strings for easier lookup
       classes_to_skip = BrokenRecord::Config.classes_to_skip.map(&:to_s)
-      ObjectSpace.each_object(Class) do |klass|
+      ActiveRecord::Base.descendants.each do |klass|
         if ActiveRecord::Base > klass
```

Comment:
@kbackowski Since you're using `ActiveRecord::Base.descendants` now, is this `if` check no longer required?